src/main/scala/rocket/PTW.scala

```diff
@@ -16,25 +16,36 @@ import freechips.rocketchip.util._
 import freechips.rocketchip.util.property
 import scala.collection.mutable.ListBuffer
 
+/** PTE request from TLB to PTW
+  *
+  * TLB send a PTE request to PTW when TLB miss and it needs a page table walk
+  */
 class PTWReq(implicit p: Parameters) extends CoreBundle()(p) {
   val addr = UInt(width = vpnBits)
   val need_gpa = Bool()
   val vstage1 = Bool()
   val stage2 = Bool()
 }
 
+/** PTE info from PTW to TLB
+  *
+  * containing: target PTE, exceptions, two-satge tanslation info */
 class PTWResp(implicit p: Parameters) extends CoreBundle()(p) {
   val ae_ptw = Bool()
   val ae_final = Bool()
```

Comment:
`ae` means access fault, I think those signals should be well documented as well.